editor.planx.uk/src/pages/FlowEditor/lib/__tests__/overrideAnswer.test.ts

```diff
@@ -2,35 +2,62 @@ import { vanillaStore } from "../store";
 
 const { getState, setState } = vanillaStore;
 
-const { overrideAnswer, currentCard } = getState();
+const { overrideAnswer, currentCard, upcomingCardIds, record } = getState();
 
 test("it clears the correct breadcrumb and navigates back to the right node", async () => {
+  // set up initial state, confirm our passport computes as expected
   setState({
     flow: flowWithPropertyTypeOverride,
     breadcrumbs: propertySearchBreadcrumb,
   });
+  const initialPassport = getState().computePassport();
+  expect(initialPassport.data).toHaveProperty(
+    ["property.type"],
+    ["residential.dwelling.house.semiDetached"]
+  );
```

Comment:
I was previously doing a lot of `expect(Object.keys(passport.data)).toContain("some.key")`, but learned that Jest's `expect().toHaveProperty()` method accepts `[...]` syntax to access keys with dots in them (whereas normal, un-bracketed behavior is to deep reference) which feels cleaner & stronger. 

https://jestjs.io/docs/expect#tohavepropertykeypath-value